features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Login/Login.js

```diff
@@ -91,79 +104,105 @@ class Login extends Component {
 
     handleUsernameChange = (event) => {
         this.setState({
-           username : event.target.value
+            username : event.target.value
```

Comment:
I think we can merge `handleUsernameChange` and `handlePasswordChange` input handlers using named input fields ?

Reply:
Documentation: [#handling-multiple-inputs](https://reactjs.org/docs/forms.html#handling-multiple-inputs)
```javascript
  handleInputChange(event) {
    const target = event.target;
    const value = target.type === 'checkbox' ? target.checked : target.value;
    const name = target.name;

    this.setState({
      [name]: value
    });
  }
```

Reply:
Implemented with the commit https://github.com/RenukaFernando/carbon-apimgt/commit/53f0bccdb13eaf943f184b0e8105477b26598223